app/controllers/tasks_controller.rb

```diff
@@ -1,72 +1,72 @@
-      elsif
-        @tasks = current_user.tasks.order(created_at: "DESC").page(params[:page]).per(2)
-      end
+    if params[:sort_expired]
+      @tasks = current_user.tasks.order(end_date: "DESC").page(params[:page]).per(2)
+    elsif params[:sort_priority]
+      @tasks = current_user.tasks.order(priority: "ASC").page(params[:page]).per(2)
+    elsif
+      @tasks = current_user.tasks.order(created_at: "DESC").page(params[:page]).per(2)
+    end
 
-      if params[:search].present?
-        if params[:name].present? && params[:status].present?
-          @tasks = current_user.tasks.name_search(params[:name]).status_search(params[:status]).page(params[:page]).per(2)
-        elsif params[:name].present?
-          @tasks = current_user.tasks.name_search(params[:name]).page(params[:page]).per(2)
-        elsif params[:status].present?
-          @tasks = current_user.tasks.status_search(params[:status]).page(params[:page]).per(2)
-        end
+    if params[:search].present?
+      if params[:name].present? && params[:status].present?
+        @tasks = current_user.tasks.name_search(params[:name]).status_search(params[:status]).page(params[:page]).per(2)
+      elsif params[:name].present?
+        @tasks = current_user.tasks.name_search(params[:name]).page(params[:page]).per(2)
+      elsif params[:status].present?
+        @tasks = current_user.tasks.status_search(params[:status]).page(params[:page]).per(2)
       end
     end
+  end
 
-    def new
-      @task = Task.new
-    end
+  def new
+    @task = Task.new
+  end
 
-    def create
-      @task = Task.create(task_params)
-      @task.user_id = current_user.id
-      if @task.save
+  def create
+    @task = Task.create(task_params)
+    @task.user_id = current_user.id
+    if @task.save
       redirect_to tasks_path, notice: ('create')
-      else
-        render :new if @task.invalid?
-      end
+    else
+      render :new if @task.invalid?
     end
+  end
 
-    def show
-      @task = Task.find(params[:id])
+  def show
+    @task = Task.find(params[:id])
 
-    end
+  end
 
-    def edit
-      @task = Task.find(params[:id])
+  def edit
+    @task = Task.find(params[:id])
 
-    end
+  end
 
-    def update
-      @task = Task.find(params[:id])
+  def update
+    @task = Task.find(params[:id])
 
-      if @task.update(task_params)
-        redirect_to tasks_path, notice: ('update')
-      else
-        render :edit
-      end
+    if @task.update(task_params)
+      redirect_to tasks_path, notice: ('update')
+    else
+      render :edit
     end
+  end
 
-    def destroy
-      @task = Task.find(params[:id])
+  def destroy
+    @task = Task.find(params[:id])
 
-      @task.destroy
-      redirect_to tasks_path, notice: ('destroy')
-    end
-
-    private
-    def task_params
-      params.require(:task).permit(:name, :detail, :end_date, :priority, :status, :user_id)
-    end
+    @task.destroy
+    redirect_to tasks_path, notice: ('destroy')
+  end
 
+  private
+  def task_params
```

Comment:
少し細かいですが、privateの下には1行改行を入れるのが一般的かと思いました！
```suggestion
  private
  
  def task_params
```

Reply:
56e44db で確認しました。